src/abbreviations.jl

```diff
@@ -281,6 +281,116 @@ function format(::MethodList, buf, doc)
 end
 
 
+
+
+
+
+
+
+#
+# `TypedMethodList`
+#
+
+"""
+The singleton type for [`TYPEDMETHODLIST`](@ref) abbreviations.
+
+$(:FIELDS)
+"""
+struct TypedMethodList <: Abbreviation end
+
+"""
+An [`Abbreviation`](@ref) for including a list of all the methods that match a documented
+`Method`, `Function`, or `DataType` within the current module.
+
+# Examples
+
+The generated markdown text will look similar to the following example where a function
+`f` defines two different methods (one that takes a number, and the other a string):
+
+````markdown
+```julia
+f(x::Int)
+```
+
+defined at [`<path>:<line>`](<github-url>).
+
+```julia
+f(y::String)
+```
+
+defined at [`<path>:<line>`](<github-url>).
+````
+"""
+const TYPEDMETHODLIST = TypedMethodList()
+
+function format(::MethodList, buf, doc)
+    local binding = doc.data[:binding]
+    local typesig = doc.data[:typesig]
+    local modname = doc.data[:module]
+    local func = Docs.resolve(binding)
+    local groups = methodgroups(func, typesig, modname; exact = false)
+    if !isempty(groups)
+        
+        #START OF COPIED CODE FROM TYPEDMETHODSIGNATURES
```

Comment:
@unnamedunknownusername could this code just be refactored out into a reusable function rather than copying it across?

Reply:
@MichaelHatherly Have seperated out repeated code into a resusable function let me know if I have missed anything